### PR TITLE
Check that at least one of the Eth ports is fast enough

### PR DESCRIPTION
**Pull request by pyhys**
Fix error setting mechanism in case of too slow ports

### CODE REVIEW

File: src/device/pf_pdport.c [hunk 1]
```diff
@@ -240,6 +240,50 @@ int pf_pdport_save_all (pnet_t * net)
...
+#if LOG_DEBUG_ENABLED(PNET_LOG)
+   const pnet_port_cfg_t * port_config;
+#endif
+
+   pf_port_init_iterator_over_ports (net, &port_iterator);
+   loc_port_num = pf_port_get_next (&port_iterator);
+   while (loc_port_num != 0)
+   {
+      eth_status = pf_pdport_get_eth_status (net, loc_port_num);
+      fast_enough = eth_status.operational_mau_type >=
```
Comment (olbjo) [inline src/device/pf_pdport.c:259]: Maybe only plugged / running interfaces shall be included in check? 
In a multiport scenario I unplugged interfaces/ports could return a default value .

**Review by olbjo — COMMENTED** [review]
Looks good, one comment to check.

File: src/device/pf_pdport.c (hunk 1 above)
Comment (pyhys) [reply src/device/pf_pdport.c:259]: We are looking if there is at least one, so there is no harm in looking at all interfaces (to keep logic simple).
I avoid returning early, just to have the debug log show the speed of all interfaces.

Comment (pyhys) [reply src/device/pf_pdport.c:259]: Updated to return true if MAU-type is fast enough and the interface is up.

**Review by olbjo — APPROVED** [review]
Great!

**Pull request merged**